api/nodes/filters.py

```diff
@@ -55,11 +55,19 @@ def build_query_from_field(self, field_name, operation):
             return Q(root__guids___id__in=operation['value'])
 
         if field_name == 'preprint':
-            not_preprint_query = (
-                Q(preprint_file=None) |
-                Q(_is_preprint_orphan=True) |
-                Q(_has_abandoned_preprint=True)
-            )
+            # required for "All my preprints" section on the "My Projects" page until preprint/node divorce
+            if utils.is_truthy(self.request.query_params.get('include_unpublished_preprints')):
```

Comment:
Hmm, so `?filter[preprint]=false&include_unpublished_preprints=true` would not include nodes with abandoned preprints. That seems fine, and like something no one will ever notice.